### PR TITLE
eSRM script update

### PR DESCRIPTION
**Pull request by VictoriaDel**

### CODE REVIEW

File: scripts/sem_bleaching_map.py [hunk 1]
```diff
@@ -45,64 +54,92 @@
...
+        self._spotsimage = None
+
+        # scale is the distance between the SR pixels in SEM pixels at ebeam.scale=1
+        self.scale = (pxs / self.ebeam.pixelSize.value[0], pxs / self.ebeam.pixelSize.value[1])
+
         lprng = self.light.power.range
-        if not (lprng[0] < lpower < lprng[1]):
-            raise ValueError("starting light power value must be between %s" % (lprng,))
+        if not (lprng[0] <= lpower <= lprng[1]):
+            raise ValueError("starting light power value must be between %s", lprng)
```
Comment (pieleric) [inline scripts/sem_bleaching_map.py:122]: The previous line was correct ->  `% (lprng,)`

File: scripts/sem_bleaching_map.py [hunk 2]
```diff
@@ -1,33 +1,42 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-'''
+"""
 Created on 20 Mar 2014
 
-@author: Éric Piel
+@author: Éric Piel, modified by Aaro Vakevainen, modified by Lennard Voortman
```
Comment (Kleijwegt) [inline scripts/sem_bleaching_map.py:6]: Who are Aaro Vakevainen and Lennard Voortman? Maybe we can write there affiliation behind there name in brackets.
@VictoriaDel Shouldn't your name also be here/

File: scripts/sem_bleaching_map.py [hunk 3]
```diff
@@ -249,28 +304,25 @@ def configure_ccd(self, roi):
...
 
         logging.info("CCD res = %s, binning = %s",
-                      self.ccd.resolution.value,
-                      self.ccd.binning.value)
+                     self.ccd.resolution.value,
+                     self.ccd.binning.value)
+
+        # turn on cooling of the scmos
+        # self.ccd.fanSpeed.value = 1
+        # self.ccd.targetTemperature = 0
```
Comment (Kleijwegt) [inline scripts/sem_bleaching_map.py:324]: Do these lines need to stay commented out? Or can they be deleted?

File: scripts/sem_bleaching_map.py [hunk 4]
```diff
@@ -309,137 +360,331 @@ def configure_sem_for_tile(self):
...
+
+        # the center of the border is offset by half the roi, and half the border
+        px_border_trans = [px_width[0]/2 + b_px[0]/2 * ebeam_scale[0],
+                           px_width[1]/2 + b_px[1]/2 * ebeam_scale[1]]  # in px when scale=1
+        px_roi_width = [int(px_width[0]/ebeam_scale[0]),
+                        int(px_width[1]/ebeam_scale[1])]  # in px at current scale
+
+        # The borders are acquired in the following order
+        #
+        #  11111111111
```
Comment (Kleijwegt) [inline scripts/sem_bleaching_map.py:538]: Nice "picture"!

File: scripts/sem_bleaching_map.py [hunk 5]
```diff
@@ -463,101 +720,210 @@ def estimate_acq_time(self, shape):
...
 
             # Let's go!
-            sem_data = []
-            # start with the original fluorescence count (no bleaching)
+            logging.info("Acquiring full image 0")
+            self.acquire_and_save_optical_survey(fn, "fullBefore")
+
+            # logging.info("Starting with bleaching a border around the ROI")
+            # sem_border = self.bleach_borders(roi)
+            # self.save_data(sem_border,"%s-SEM_bleached_border%s" % (fn_base, fn_ext))
```
Comment (Kleijwegt) [inline scripts/sem_bleaching_map.py:799]: Is this code supposed to be commented out?

File: scripts/sem_bleaching_map.py [hunk 6]
```diff
@@ -463,101 +720,210 @@ def estimate_acq_time(self, shape):
...
+                    overhead_est = 0
+                    # overhead_est = 1 * dur_frame + 0.5
+
+                    # self.ccd.exposureTime.value = dur_frame + overhead_est
+                    # logging.info("Setting exposuretime for spotsimage to: %s (includes %s overhead)",
+                    # units.readable_time((dur_frame + overhead_est)),
+                    # units.readable_time((overhead_est)))
+                    # self.ccd_acq_complete.clear()
+                    # self.ccd.data.subscribe(self.on_ccd_data)
+                    # time.sleep(0.2)
```
Comment (Kleijwegt) [inline scripts/sem_bleaching_map.py:859]: Is this code supposed to be commented out? If it has functionality maybe write a comment above it with the purpose of it

File: scripts/sem_bleaching_map.py (hunk 2 above)
Comment (pieleric) [reply scripts/sem_bleaching_map.py:6]: When working on this script, Lennard used to work in Delmic, while Aaro was a TUDelft student (I think).

File: scripts/sem_bleaching_map.py (hunk 6 above)
Comment (VictoriaDel) [reply scripts/sem_bleaching_map.py:859]: This is not written by me, so I didn't want to delete it. People that know how to use the script (like phd students in TU-Delft) use to comment and uncomment some parts of the script. Anyway, I will have to delete this pull-request and make a new one :)

**Pull request closed without merging**